xbmc/guilib/GUIWindowManager.cpp

```diff
@@ -1196,6 +1196,14 @@ CGUIWindow* CGUIWindowManager::GetWindow(std::type_index type) const
   return NULL;
 }
 
+CGUIDialog* CGUIWindowManager::GetDialog(int id) const
+{
+  CGUIWindow *window = GetWindow(id);
+  if (window->IsDialog())
```

Comment:
I'm hitting a segfault here with window=null. Changing to 
```if (window && window->IsDialog())```
avoids the crash. See: http://forum.kodi.tv/showthread.php?tid=310869